packages/preview-middleware-client/src/adp/ui/AddFragment.view.xml

```diff
@@ -23,14 +23,25 @@
                             <core:Item key="{key}" text="{value}" />
                         </ComboBox>
                         <Label text="Index" />
-                        <ComboBox
-                            selectedKey='{/selectedIndex}'
-                            items="{
-                                path: '/index',
-                                sorter: { path: 'text' }
-                            }">
-                            <core:Item key="{key}" text="{value}" />
-                        </ComboBox>
+                        <HBox alignItems="Center">
+                            <ComboBox
+                                enabled="{/indexHandlingFlag}"
+                                selectedKey='{/selectedIndex}'
+                                items="{
+                                    path: '/index',
+                                    sorter: { path: 'text' }
+                                }">
+                                <core:Item key="{key}" text="{value}" />
+                            </ComboBox>
+                            <core:Icon
+                                useIconTooltip="false"
+                                visible="{/specialIndexHandlingIcon}" 
+                                src="sap-icon://message-information" 
+                                class= "sapUiTinyMarginBegin" 
+                                tooltip="{/iconTooltip}"
+                                >
+                            </core:Icon>
```

Comment:
The icon can closed because it does not have any children. <Icon />

Reply:
updated in [37cc141](https://github.com/SAP/open-ux-tools/pull/1477/commits/37cc141c0f4e040454ed96e66536e4c64f27436b)